src/App.tsx

```diff
@@ -15,36 +16,107 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  SORT_ALPHABETICALLY = 'Sort alphabetically',
+  SORT_BY_LENGTH = 'Sort by length',
```

Comment:
Enum properties shouldn't be in upper-case https://mate-academy.github.io/fe-program/js/extra/typescript
![image](https://github.com/user-attachments/assets/b1b0505b-6336-4c23-9294-ce3d884c1d99)